network/src/commonMain/kotlin/com/wire/kalium/network/api/v4/authenticated/MessageApiV4.kt

```diff
@@ -20,9 +20,70 @@ package com.wire.kalium.network.api.v4.authenticated
 
 import com.wire.kalium.network.AuthenticatedNetworkClient
 import com.wire.kalium.network.api.base.authenticated.message.EnvelopeProtoMapper
+import com.wire.kalium.network.api.base.authenticated.message.MessageApi
+import com.wire.kalium.network.api.base.authenticated.message.QualifiedSendMessageResponse
+import com.wire.kalium.network.api.base.model.ConversationId
 import com.wire.kalium.network.api.v3.authenticated.MessageApiV3
+import com.wire.kalium.network.exceptions.SendMessageError
+import com.wire.kalium.network.serialization.XProtoBuf
+import com.wire.kalium.network.utils.NetworkResponse
+import com.wire.kalium.network.utils.wrapKaliumResponse
+import io.ktor.client.call.body
+import io.ktor.client.request.parameter
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
 
 internal open class MessageApiV4 internal constructor(
     authenticatedNetworkClient: AuthenticatedNetworkClient,
-    envelopeProtoMapper: EnvelopeProtoMapper
-) : MessageApiV3(authenticatedNetworkClient, envelopeProtoMapper)
+    private val envelopeProtoMapper: EnvelopeProtoMapper
+) : MessageApiV3(authenticatedNetworkClient, envelopeProtoMapper) {
+
+    override suspend fun qualifiedSendMessage(
```

Comment:
So this was not implemented on V0? What's does it implementation on V0 do? Shouldn't we return not supported error like we do on other endpoints?